app/data/doc-collections/account-linking.json

```diff
@@ -17,25 +17,25 @@
           {
             "title": "Account Model",
             "tags": ["guide", "account-linking"],
-            "description": "Learn about the account model enables through Account Linking",
-            "href": "/account-linking/guides/account-model"
+            "description": "Learn about the account model that is enabled through Account Linking",
+            "href": "guides/account-model"
```

Comment:
Is this correct, since the current path does include `account-linking`?

Reply:
Yes correct, relative paths work and I think more readable. Starting with a slash means absolute path which can also work. Other configs use relative paths so I'd like to be consistent